client/src/pages/OpeningDashboardPage/AddSlotButton.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const AddSlotButton = () => {
+  return (
+    <Link to="./create">
+      <button
+        type="button"
+        className="focus:outline-none text-white bg-green-700 hover:bg-green-800 focus:ring-4 focus:ring-green-300 font-medium rounded-lg text-sm px-5 py-2.5 me-2 mb-2 dark:bg-green-600 dark:hover:bg-green-700 dark:focus:ring-green-800"
```

Comment:
the style as className shouldn't be this long, you can use className "filled-btn" I defined for button, take a look in index.css for more detail